src/components/pages/Map/map.css

```diff
@@ -1,8 +1,17 @@
+
 .marker-btn {
   width: 15px;
   height: 20px;
 }
-
+.cluster-marker {
+  color: #fff;
+  background: #1978c8;
+  border-radius: 50%;
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
 /* .marker-btn:hover {
```

Comment:
This commented out code should be taken out.

Reply:
changes committed